terminus-ui/paginator/testing/src/test-helpers.ts

```diff
@@ -0,0 +1,37 @@
+import { ComponentFixture } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { selectOption } from '@terminus/ui/select/testing';
+
+
+export type TsPaginatorPage
+  = 'first'
+  | 'last'
+  | 'next'
+  | 'previous'
+;
+
+/**
+ * Helper function to click a button to update results set
+ *
+ * @param fixture - fixture
+ * @param dir - direction: first, last, next or previous, to determine which button to click
+ */
+// tslint:disable-next-line no-any
+export function clickForResults(fixture: ComponentFixture<any>, dir: TsPaginatorPage) {
```

Comment:
Perhaps there's a more clear name for this helper since it's only focused on clicking a page button. (thinking about consumers quickly understanding why/how they should use this)